test/integration/idle_timeout_integration_test.cc

```diff
@@ -104,7 +113,8 @@ TEST_P(IdleTimeoutIntegrationTest, PerStreamIdleTimeoutHeadRequestAfterDownstrea
   EXPECT_FALSE(upstream_request_->complete());
   EXPECT_EQ(0U, upstream_request_->bodyLength());
   EXPECT_TRUE(response->complete());
-  EXPECT_STREQ("408", response->headers().Status()->value().c_str());
+  EXPECT_STREQ(std::to_string(enumToInt(Http::Code::RequestTimeout)).c_str(),
```

Comment:
Consider assigning this first parameter to a class/file constant

Reply:
+1 -- I just felt a symbolic name was more readable than "408".

You also don't need to take .c_str() on either side -- just compare as strings.